code/modules/mob/living/carbon/superior_animal/lodge/types/cerberus.dm

```diff
@@ -3,7 +3,7 @@
 	name = "cerberus"
 	desc = "A domesticated hell diver kept fat, happy, and loyal by the local hunting lodge that breed them as hunting allies and guard animals. Favored especially for their asexual \
 	reproduction after being fed various mushrooms, a fact that baffles some soteria scientists. Unlike a standard hell diver the cerberus isn't as \
-	obsessed with food but gladly lets itself get pulled along. The instinct to attack insectile creatures that makes these attempt to maul cht'mants to death, has been bred out of them."
+	obsessed with food but gladly lets itself get pulled along. Special training allows for them to follow anyone who states 'Cerberus Follow', while 'Cerberus Stop' will make them halt."
```

Comment:
Again, now it just need to have the creature's name & Follow/Stop in the same sentence